.travis.yml

```diff
@@ -8,6 +8,7 @@ aliases:
   - &cocoapods
     script:
       - rm .swift-version
+      - gem install cocoapods
```

Comment:
I think CocoaPods is pre-installed on Travis (on `language: objective-c`).

Reply:
Do you mean upgrading CocoaPods? If so `gem update cocoapods` should be appropriate.

Reply:
Yup, it looks [works](https://travis-ci.org/Carthage/Commandant/jobs/502880236#L663-L674) as I expected.